test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -68,6 +70,11 @@ export default class PossibleIssues extends Component {
                 const issueCreator = <a href={relatedIssues.items[index].user.html_url} target="_blank" rel="noopener noreferrer">{relatedIssues.items[index].user.login}</a>;
                 const createdAt = new Date(relatedIssues.items[index].created_at).toLocaleString();
                 const issueState = relatedIssues.items[index].state;
+                const userFeedback = <>
+                <Button ><SmileOutlined style={{fontSize: '25px', color: 'green'}} /></Button> 
```

Comment:
For here, you can add linking button to a function, with passing info (e.g. repoName, issue, issueCreator) as the inputs of the function here. Note that an extra input for `smile` or `frown` is needed for the function.